src/main/java/org/isf/dicom/model/FileDicom.java

```diff
@@ -50,6 +50,8 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
+import com.drew.lang.annotations.Nullable;
```

Comment:
This import is not needed and is unused.

Reply:
Thanks!